packages/EasyStandard/tests/Rector/ExplicitBoolCompareRector/Fixture/FuncCallWithShortTrueComparison.php.inc

```diff
@@ -0,0 +1,12 @@
+<?php
+if (\in_array('value', [])) {
+    echo 'in array';
+}
+?>
+
+-----
+<?php
```

Comment:
We don't need the second part for correct cases.

Reply:
Done.